### PR TITLE
Start implementing working pipeline

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
cc @kegl @aboucaud @gautiernguyen

**Comment by kegl** [comment]
Isn't log loss what we have here? https://github.com/paris-saclay-cds/ramp-workflow/blob/master/rampwf/score_types/negative_log_likelihood.py

**Comment by jorisvandenbossche** [comment]
> Isn't log loss what we have here? https://github.com/paris-saclay-cds/ramp-workflow/blob/master/rampwf/score_types/negative_log_likelihood.py

Yes, that might be. From a quick look at the code, the difference seems to be that the version in ramp is taking the mean of the scores, while in sklearn they take the sum.

**Comment by jorisvandenbossche** [comment]
Sklearn version:

https://github.com/scikit-learn/scikit-learn/blob/acf3babfe5aa16e3189f1d2f7d9ffe45357630a8/sklearn/metrics/classification.py#L1815-L1819

**Comment by kegl** [comment]
I added nll, it seems it's exactly the same. Does the sklearn implementation protect against stupid input (unnormalized, outside [0,1])?

**Comment by kegl** [comment]
OK, I put our nll in the scores, we should have this discussion in rampwf, independently of this ramp.

**Comment by jorisvandenbossche** [comment]
> Does the sklearn implementation protect against stupid input (unnormalized, outside [0,1])?

As far as I can see, it clips and normalizes and has an eps of 1e-15 (http://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html)

File: submissions/starting_kit/feature_extractor.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return self
+
+    def transform(self, X_df):
+        return _transform(X_df)
+
+    # def transform(self, X_df):
+    #     """Compute the El Nino mean at time t - (12 - X_ds.n_lookahead).
+
+    #     Corresponding the month to be predicted.
+    #     """
```
Comment (aboucaud) [inline submissions/starting_kit/feature_extractor.py]: Comments to be removed

File: submissions/starting_kit/feature_extractor.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    #     X_array = enso_valid.reshape((-1, 1))
+    #     return X_array
+
+
+@memory.cache
+def _transform(X_df):
+    X_df_new = X_df.copy()
+    #X_df_new = computeBeta(X_df_new)
+    #X_df_new = computeRollingStd(X_df_new, '15min', 'Beta')
+    X_df_new = computeRollingStd(X_df_new, '2h', 'Beta')
```
Comment (aboucaud) [inline submissions/starting_kit/feature_extractor.py]: Some testing lines to be removed
